modules/ROOT/pages/maintenance/backup.adoc

```diff
@@ -0,0 +1,44 @@
+= Backup
+:toc: right
+
+:description: Making a backup of Infinite Scale is an important task that should be done on a regular basis.
+
+== Introduction
+
+{description} This ensures that in case of issues, data can be restored from a backup.
+
+== General Considerations
+
+Infinite Scale can run in two different setups when it comes to storing data.
+
+* All data, means configuration, user data, and metadata is stored on POSIX compliant filesystems.
+* User data is stored as blob on an S3 compliant storage and configuration respectively metadata is stored on POSIX
+
+For details on supported filesystems see: xref:prerequisites/prerequisites.adoc#filesystems-and-shared-storage[Filesystems and Shared Storage]
+
+Depending on how this data is stored, backup strategies and tasks will differ, see the sections below.
+
+For the time being, you have to fully stop the Infinite Scale instance to create a consistant state necessary for taking the backup. In the scenarious shown, this consistancy state is a prerequisite for all upcoming tasks and the way to it is not explicit mentioned anymore. When the backup has finished, the Infinite Scale instance can be started again.
+
+Note that taking a backup can be a task taking only a view seconds if the underlaying filesystem respectively storage is able to provide snapshot capabilities. After taking the necessary snapshots, the Infinite Scale instance can be started again. Based on the snapshots, any physical copy of data to a secondary storage can run independently by a follow up task.
```

Comment:
```suggestion
Note that creating a backup may only take a view seconds if the underlying filesystem of your storage provides snapshot capabilities. After taking the necessary snapshots, the Infinite Scale instance can be started again. To prevent data loss in case of hardware failure, we recommend copying the snapshots to a secondary storage.
```